docs/source/conf.py

```diff
@@ -86,15 +88,15 @@
 #
 # This is also used if you do content translation via gettext catalogs.
 # Usually you set "language" from the command line for these cases.
-language = None
```

Comment:
This is the only line that needed changing, but my pre-commit has an attitude problem.